modules/social_features/social_album/social_album.install

```diff
@@ -12,9 +12,44 @@ use Drupal\user\RoleInterface;
  * Implements hook_install().
  */
 function social_album_install() {
-  // Set some default permissions.
-  _social_album_set_permissions();
-  _social_album_set_group_permissions();
+  // Grant the default permissions for this feature.
+  user_role_grant_permissions(
+    RoleInterface::AUTHENTICATED_ID,
+    [
+      'view node.album.field_content_visibility:public content',
```

Comment:
Good questions.

Since previously Anonymous user was not able even to see public content for the album I think we either leave `'view node.album.field_content_visibility:public content'` for unverified user or remove all permissions for albums for him.

Same for other content (e.g. Books)